uber/decorators.py

```diff
@@ -182,7 +182,7 @@ def check_dept_admin(session, department_id=None, inherent_role=None):
 def assert_dept_admin(session, department_id=None, inherent_role=None):
     message = check_dept_admin(session, department_id, inherent_role)
     if message:
-        raise HTTPRedirect("../common/invalid?message={}", message)
+        raise HTTPRedirect("../landing/invalid?message={}", message)
```

Comment:
FWIW, of all of the nice renames, I think renaming "common" to "landing" is the best one.  Even I can never remember what "common" mean.